features/in_onderzoek.feature

```diff
@@ -166,188 +165,283 @@ Functionaliteit: in onderzoek
       | burgerservicenummer | 555550001                       |
-      | fields              | ouders.<veld>                   |
-      Dan heeft de ouder met ouderAanduiding '<ouder>' de volgende 'inOnderzoek' gegevens
-      | naam                             | waarde       |
-      | <veld>                           | true         |
-      | datumIngangOnderzoek.datum       | 2022-03-07   |
-      | datumIngangOnderzoek.type        | Datum        |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
-      En heeft de ouder met ouderAanduiding '<andere ouder>' GEEN 'inOnderzoek' gegevens
-      En heeft de persoon GEEN 'inOnderzoek' gegevens
+      | fields              | ouders                          |
+      Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
+      | naam                                         | waarde       |
+      | ouderAanduiding                              | <ouder>      |
+      | inOnderzoek.<veld>                           | true         |
+      | inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoek.type        | Datum        |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      En heeft de persoon een 'ouder' met de volgende gegevens
+      | naam            | waarde         |
+      | ouderAanduiding | <andere ouder> |
 
       Voorbeelden:
       | waarde | veld                                    | ouder | andere ouder |
-      | 020220 | burgerservicenummer                     | 1     | 2            |
+      | 020120 | burgerservicenummer                     | 1     | 2            |
       | 030410 | geslacht                                | 2     | 1            |
       | 026210 | datumIngangFamilierechtelijkeBetrekking | 1     | 2            |
 
     Abstract Scenario: persoon heeft <groep> <veld> van partner in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                        | waarde    |
-      | burgerservicenummer         | 555550001 |
-      En de persoon heeft een partner met de volgende gegevens
       | naam                | waarde    |
-      | burgerservicenummer | 555550002 |
-      En de partner met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                     | waarde    |
+      | burgerservicenummer                      | 555550002 |
+      | aanduiding gegevens in onderzoek (83.10) | <waarde>  |
+      | datum ingang onderzoek (83.20)           | 20220307  |
+      En de 'partner' heeft de volgende 'naam' gegevens
+      | naam                  | waarde        |
+      | geslachtsnaam (02.40) | geslachtsnaam |
```

Comment:
De waarde 'geslachtsnaam' is geen reële naar, wijzig dit in bijv. 'Groenen'. Deze wordt overigens verder niet gebruikt in dit scenario.

Reply:
#1202